docs/source/commands/configtxgen.md

```diff
@@ -20,7 +20,7 @@ Usage of configtxgen:
   -asOrg string
     	Performs the config generation as a particular organization (by name), only including values in the write set that org (likely) has privilege to set
   -channelCreateTxBaseProfile string
-    	Specifies a profile to consider as the orderer system channel current state to allow modification of non-application parameters during channel create tx generation. Only valid in conjunction with 'outputCreateChannelTx'.
+    	[DEPRECATED] Specifies a profile to consider as the orderer system channel current state to allow modification of non-application parameters during channel create tx generation. Only valid in conjunction with 'outputCreateChannelTx'.
```

Comment:
If this is deprecated why not remove it altogether?

Reply:
I think we want to maintain backwards compatibility? i.e. be able to manage with the new version of the tool old deployments?